### ISSUE TITLE
VideoMAE vs DropMAE

### ISSUE THREAD

**Issue by laisimiao** [jimmy-dq/DropMAE#2]
I noticed someone earlier pretrain the vit using masked autodecoder on video data: eg. VideoMAE and VideoMAEv2. Do you think that pretrained vit could also be finetuned in downstream vot and vos tasks. And it has the similiar spirit with your work?

**Comment by jimmy-dq** [comment]
Hi, our baseline TwinMAE is a simple extension of VideoMAE, which keeps consistent input representations (frame pairs) w/ the downstream VOT/VOS tasks.  Yes, I believe it's helpful to use VideoMAE for further VOT/VOS fine-tuning, but VideoMAE uses cube embedding and clip temporal downsampling, which is different from our downstream task architectures. If you do so, better to adjust the downstream tasks accordingly.
